kong/router/expressions.lua

```diff
@@ -19,8 +24,35 @@ local PROTOCOLS_OVERRIDE = {
 }
 
 
-local function get_exp_and_priority(route)
+-- net.port => net.dst.port
+local function transform_expression(route)
```

Comment:
How about naming this function "migrate_deprecated_variables"?

Reply:
It will have more transform actions in the future, not only for migration.